app/assets/javascripts/greetings.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
Similar to the comment below, this file doesn't seem to be needed. However, I guess you might have removed it as part of the PR which gets rid of coffeescript, so feel free to skip this one.